src/content/experimental/layout/layout.md

```diff
@@ -1,24 +1,283 @@
+<page-intro>
+The 2x Grid is the geometric foundation of all the visual elements of IBM Design, from typography to columns, boxes, icons, and illustrations. It provides structure and guidance to creative decision-making.
+ </page-intro>
 
-## Fixed 'Mini Unit' Grid
-Mini unit is the basic of the 2x Grid. It is the fundamental and the guide of every component. All mini units are 8 pixel by 8 pixel. Paddings, gutters, margins, spacers, and the dimensions of shapes are determined by mini units.
+1.  [Mini Unit](#mini-unit)
 
-![mini unit grid](images/mini_unit_grid.png)
+2.  [2x Grid Fundamentals](#2x-grid-fundamentals)
 
-## Fluid 2x Grid
-### Columns
-Column structure is important for layouts. The primary layout choices for designs are 1, 2, 4, 8, and 16 columns. Maintaining column structure is vital to achieving a well crafted duo experience.
+4.  [Grid Behaviors](#grid-behaviors)
+
+5.  [Screen Regions](#screen-regions)
+
+6.  [Sizing Scale](#sizing-scale)
+
+
+## Mini Unit
+
+The basic unit of 2x Grid geometry is the 8-pixel square Mini Unit. Multiples of mini units compose the dimensions of columns, rows, boxes, and their margins and padding. The mini unit adapts to your content while maintaining a consistent visual rhythm.
+
+![mini unit grid](images/mini_unit_grid_boarder.png)
+
+At [breakpoint boundaries](#breakpoints), mini units divide the screen into a fixed master grid, and multiples of mini units map to [fluid grid](#fluid-grid) column widths and row heights. [Fixed boxes](#fixed) are sized in mini units, as are the fixed dimension of [hybrid boxes](#hybrid).
+
+[Margin](#margin) and [padding](#padding) are always applied in fixed mini unit multiples. Boxes are sized by applying a [sizing scale](#sizing-scale) based on either fixed mini units or fluid column widths, along with a core set of [aspect ratios](#aspect-ratios).
+
+
+## 2x Grid Fundamentals
+
+The core concept of the 2x Grid is to divide or multiply by two, forming a visual rhythm. Fluid grids are built by division, whereas fixed grids are built with multiplication. On breakpoint boundaries, the sizes match. Margins and padding are applied consistently in both cases.
+
+1.  [Columns and Rows](#columns-and-rows)
+2.  [Margins](#margins)
+3.  [Padding](#padding)
+4.  [Gutters](#gutters)
+5.  [Breakpoints](#breakpoints)
+
+### Columns and Rows
+
+Columns and rows create key lines that are essential for visual rhythm, especially for typography. Construct columns by dividing a space into a fluid grid, or by tiling fixed boxes in multiples.
+
+![grid columns](images/2x_grid_columns_new.png)
+
+A [fluid grid](#fluid-grid) divides space by two. This division is immediately evident in the 2x column structure. Within a breakpoint, column count is constant and unit size scales with screen size. Choose a column count by starting with one, then divide by two as needed:
+
+| Fluid Grid Columns
+| -------------------
+| 1
+| 2
+| 4
+| 8
+| 16
+
```

Comment:
Did you intend the above to be a table? Its currently displaying like this 
<img width="490" alt="screen shot 2018-09-26 at 8 46 31 am" src="https://user-images.githubusercontent.com/2753488/46084196-b8716980-c168-11e8-803e-8aa5a4e54834.png">